src/layout/Likert/RepeatingGroupsLikertContainer.tsx

```diff
@@ -108,13 +108,14 @@ export const RepeatingGroupsLikertContainer = ({ node }: RepeatingGroupsLikertCo
             aria-labelledby={(hasTitle && titleId) || undefined}
             aria-describedby={(hasDescription && descriptionId) || undefined}
             className={classes.likertTable}
+            role='group'
           >
             <Table.Head
               id={`likert-table-header-${id}`}
               className={classes.likertTableHeader}
             >
               <Table.Row>
-                <Table.HeaderCell>
+                <Table.HeaderCell aria-hidden='true'>
```

Comment:
Hiding the header cells from screen readers, as they are confusing to the screen reader user. They are merely a visual aide for visual users